src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -443,24 +440,27 @@ private String encrypt(String clearValue) {
   }
 
   /**
-   * Make Case Service request to return cases by UPRN
+   * Make Case Service request to return cases by a given Sample attribute
```

Comment:
javadoc is wrong, since its a DB request. Since its a private method, why not remove all this javadoc ?